crates/python/README.md

```diff
@@ -8,6 +8,10 @@ to compile and run Quil programs on Rigetti quantum processors. Internally, it i
 While this package can be used directly, [pyQuil](https://pypi.org/project/pyquil/) offers more functionality and a 
 higher-level interface for building and executing Quil programs. This package is still in early development and breaking changes should be expected between minor versions.
 
+# Documentation
+
+Documentation for the current release of `qcs_sdk` is published [here](https://rigetti.github.io/qcs-sdk-rust/qcs_sdk.html). Every version of `quil` ships [with type stubs](https://github.com/rigetti/qcs-sdk-rust/tree/main/crates/python/qcs_sdk) that provide type hints and documentation to Python tooling and editors that support the [Language Server Protocol](https://microsoft.github.io/language-server-protocol/) or similar.
```

Comment:
does it make sense to mention LSP here? that is, LSP doesn't directly support python type hints. So it's just the python tooling that cares about the hints, whether or not it uses LSP

Reply:
My thinking was that "Python tooling and editors" implies both LSP servers and clients. So support could mean supporting as a client or implementing as a server. I tried to keep it broad because a lot of modern editors use LSP in a transparent way (e.g. VSCode) so many users don't even know they are using it, and other editors use a custom solution (e.g. JetBrains). I mostly wanted to express we that (1) we have type stubs and (2) a convenient way to work with the library is to figure out how to get them in your editor.

Reply:
Per-offline discussion I trimmed down this statement to generally reference that Python tooling and editors can use the type hints.